lbz/exceptions.py

```diff
@@ -13,13 +13,14 @@
 class LambdaFWException(Exception):
     """
     Standarised for AWS Lambda exception class.
+    Log Levels: CRITICAL, ERROR, WARNING, INFO, DEBUG
```

Comment:
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings
> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.

You skipped a blank line 😉

Reply:
What does it mean? The exception itself does not contain logging functionality 🤔 

Also, the `logger` that is declared above can be removed.